README.md

```diff
@@ -77,22 +77,22 @@
 I felt the lack of an open source app of the jewish library, for PCs.
 
 
-Torat Ememt is old and no longer maintained, and Sefaria's app is great, however, it does not work well on computers.
+Torat Emet is old and no longer maintained, and Sefaria's app is great, however, it does not work well on computers.
 
-So i decided to make one myself. I did not know Dart and Flutter at all in the beginning, but it was fun. I **love** to learn new technologies!
+So I decided to make one myself. I did not know Dart and Flutter at all in the beginning, but it was fun. I **love** to learn new technologies!
 
 The database itself is accessible to everyone following the important work of the Sefaria organization, so a big thank to them for that.
 
 Key features of the project:
-* the software is FREE and will be ALLWAYS FREE.
+* the software is FREE and will remain ALWAYS FREE.
```

Comment:
That line is still incorrect, but it would ruin the meaning if I'll change it to "and will ALWAYS remain FREE"
Perhaps "and will ALWAYS REMAIN FREE"?